flocker/node/agents/test/test_blockdevice.py

```diff
@@ -3274,3 +3390,132 @@ def test_shrink(self):
         size of the filesystem.
         """
         self._resize_test(0.5)
+
+
+class AllocatedSizeTypeTests(SynchronousTestCase):
+    """
+    Tests for type coercion of parameters supplied to
+    ``allocated_size``.
+    """
+    def test_allocation_unit_bytes(self):
```

Comment:
This really shouldn't be part of the interface. Floats are the only thing that is worth worrying about. So if it happens to work that's OK, but I don't think it's worth testing.